ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -260,11 +260,11 @@ then
 		#
 		# Check if PX4IO present and update firmware if needed
 		#
-		if [ -f /etc/extras/px4io-v2_default.bin ]
+		if [ -f /etc/extras/px4io-v2.bin ]
```

Comment:
It would be great if we could avoid the monster PRs in the future - this is clearly unrelated to this PR.